vta/python/vta/top/vta_conv2d.py

```diff
@@ -501,5 +501,10 @@ def __str__(self):
 }
 
 for idx in RESNET:
-    scheds = find_schedules(RESNET[idx], vt_only=True, best_only=True)[0]
-    _WL2PLAN[RESNET[idx]] = scheds
+    f_schedules = find_schedules(RESNET[idx], vt_only=True, best_only=True)
+    if f_schedules:
+        scheds = f_schedules[0]
+        _WL2PLAN[RESNET[idx]] = scheds
+    else:
+        logging.warning("Current vta configuration not compatible with restnet18 workload!")
```

Comment:
Can we change the message to: "No valid schedule was found for the workload on current vta configuration"

Reply:
sure, already done.